website/versioned_docs/version-v0.5.0/revocation.md

```diff
@@ -104,17 +104,19 @@ The addition of revocation to IRMA is designed to be backwards compatible: when
 
 ### Scheme
 
-Revocation for a credential type is enabled in the scheme by (1) including at least one `RevocationServer` XML tag in its `description.xml`, and (2) including a *revocation attribute*:
+Revocation for a credential type is enabled in the scheme by (1) including at least one `RevocationServer` XML tag in its `description.xml`, and (2) including a separate `Attribute` with `revocation="true"` as content:
```

Comment:
may I suggest to remove `(1)` and `(2)` ? these numbers do not seem to relate to notes or subtexts or something

Reply:
You are right, this might cause confusion. I've rewritten the sentence to remove the (x) references and simplify reading.